lib/launcher.rb

```diff
@@ -149,32 +177,85 @@ def write_pid
       retry
     end
 
-    def webrick_server(app, addresses, port)
+    def add_puma_server(app, addresses, port, conn_type)
+      # IMPORTANT:
+      # The following code takes only a single host.
+      # The current reason for it, is that "run" is blocking, and in order to
+      # add support for more hosts, additional threads requires to be created
+      address = addresses.first
+      address = '0.0.0.0' if address == '*'
```

Comment:
This is IPv4 only.

Reply:
Yeah looks like webrick does listen on both: `$ curl http://[::1]:8448/features` good catch.